### PR TITLE
Namespaces protocol buffers

### PR DESCRIPTION
**Pull request by maciekrb**
All proto messages are now under the name `matrix_io`. This makes it cleaner to use them in different languages.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,47 +1,92 @@
...
+endif()
 
-# GRPC recognition_service
-PROTOBUF_GENERATE_CPP(RECOGNITION_GRPC_PROTO_SRC RECOGNITION_GRPC_PROTO_HEADER vision/recognition_service.proto)
-PROTOBUF_GENERATE_GRPC_CPP(RECOGNITION_GRPC_SRC RECOGNITION_GRPC_HEADER vision/recognition_service.proto)
-PROTOBUF_GENERATE_CPP(RECOGNITION_PROTO_SRC RECOGNITION_PROTO_HEADER vision/recognition.proto)
+# protoc grpc plugin
+find_program(GRPC_PLUGIN NAMES "grpc_${GEN_LANG}_plugin")
+if (NOT EXISTS ${GRPC_PLUGIN})
+  message(WARNING "The grpc_${GEN_LANG}_plugin plugin was not found, \
```
Comment (arhuaco) [inline CMakeLists.txt]: Make this fatal?

File: debian/libmatrixio_protos-dev.install [hunk 2]
```diff
@@ -0,0 +1,2 @@
+debian/tmp/usr/include/matrix_malos/*.h usr/include/matrix_malos
+debian/tmp/usr/include/matrix_malos/third_party/*hpp usr/include/matrix_malos/third_party
```
Comment (arhuaco) [inline debian/libmatrixio_protos-dev.install]: What goes in third_party?

File: debian/libmatrixio_protos.install [hunk 3]
```diff
@@ -0,0 +1,2 @@
+debian/tmp/usr/lib/libmatrix_malos_zmq.a usr/lib
```
Comment (arhuaco) [inline debian/libmatrixio_protos.install]: Why ZMQ here?

File: CMakeLists.txt (hunk 1 above)
Comment (maciekrb) [inline CMakeLists.txt]: We could, but then there are low level developers that only use protobuf, and may not want to compile all the gRPC stuff.

File: debian/libmatrixio_protos-dev.install (hunk 2 above)
Comment (maciekrb) [inline debian/libmatrixio_protos-dev.install]: Already corrected

File: debian/libmatrixio_protos.install (hunk 3 above)
Comment (maciekrb) [inline debian/libmatrixio_protos.install]: Already corrected

**Pull request closed without merging**